rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -252,28 +250,31 @@ def copyEncodedBoxesAndLables(self, bbox_array, label_array):
     def getEncodedBoxesAndLables(self, batch_size, num_anchors):
         return b.rocalGetEncodedBoxesAndLables(self._handle, batch_size, num_anchors)
 
-    def GetImgSizes(self, array):
+    def getImgSizes(self, array):
         return b.getImgSizes(self._handle, array)
 
-    def GetImageNameLength(self,idx):
-        return b.getImageNameLen(self._handle,idx)
+    def getImageNameLength(self, idx):
```

Comment:
What is the difference between getImageNameLen and getImageNameLength?
I think Rajy has previously asked same question

Please rename the api if needed and delete api if not used anymore